pkg/kubectl/cmd/top_node.go

```diff
@@ -98,6 +98,11 @@ func NewCmdTopNode(f cmdutil.Factory, o *TopNodeOptions, streams genericclioptio
 		}
 	}
 
+	// actually make use of the passed in streams if not already set
```

Comment:
That's weird, we are already setting these streams a few line above, there's no need for this change.

Reply:
hm, maybe the lines below address a special case where streams are not set because we pass nil `*TopNodeOptions` from a consumer of this command

Reply:
In any case, I think it'd make more sense to create a constructor for TopNodeOptions and have that constructor ensure streams are always set